docs/HACKING.md

```diff
@@ -55,7 +59,10 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
 
     $ mkvirtualenv -a $PROJECT_HOME/goldstone-server goldstone-server
 
-   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original.
+   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original. Then:
```

Comment:
They generate correct text, but to avoid confusion in the future, the "Open a new" and "Copy this" lines should be left-justified in the source document.  I.e., they're at the same level as the, "Add the following lines," line.